### DISCUSSION TITLE
Idea like - "Service temporarily unavailable. We maintanance the system. Please try later."

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#9893] [Archive]
Is it possible to disable logging in the Directus during service work? (Everyone but not admin) For example, you want to make changes to the settings and you don't want someone to log in until you fix the errors. And why? first of all - user does not get too much information or pops up an error after the made changes and we didn't have time to check new changes. The option to close the login and the option to close the transmission of data "sending them" out of Directus.
And one question I have - is such a situation possible  - when I open record and I'm working on it
and a colleague opened the same record and he wanted to change something too. Which changes will be saved, probably the latter, and what if both changes are important. How to block the possibility of editing the same record inside the application at the same time???

**Comment by rijkvanzanten** [comment]
There's currently no "maintenance mode". There's an open feature request about that here: https://github.com/directus/directus/discussions/7395. Mind adding your thoughts in there?
